pfioh/pfioh.py

```diff
@@ -818,63 +804,56 @@ def do_POST_withCompression(self, **kwargs):
             str_fileSuffix = ".zip"
 
         str_localFile   = "%s%s%s" % (str_unpackBase, str_fileOnly, str_fileSuffix)
-
+    
+        #Decoding 
         if str_encoding == "base64":
             d_ret['decode'] = {}
-            data            = base64.b64decode(fileContent)
+            d_ret['write']  = {}
             try:
-                with open(str_localFile, 'wb') as fh:
-                    fh.write(data)
+                data                        = base64.b64decode(fileContent)
                 d_ret['decode']['status']   = True
                 d_ret['decode']['msg']      = 'base64 decode successful!'
-            except:
+            except Exception as err:
                 d_ret['decode']['status']   = False
                 d_ret['decode']['msg']      = 'base64 decode unsuccessful!'
                 self.ret_client(d_ret)
                 self.qprint(d_ret, comms = 'tx')
+                self.qprint(err)
                 return d_ret
-            d_ret['decode']['timestamp']  = '%s' % datetime.datetime.now()
```

Comment:
Must have got deleted while splitting into mountdir and swiftstore. Added it back.